Cargo.toml

```
@@ -0,0 +1,38 @@
+[package]
+name = "mc-build-rs"
+version = "0.1.0"
+authors = ["MobileCoin"]
+# See https://crates.io/category_slugs for valid categories
+categories = ["development-tools::build-utils", "development-tools::cargo-plugins", "command-line-utilities"]
```

This is just a build script utilities, I assume a `cargo-plugin` is something like `cargo-sort` or `cargo-crev`. We're also not providing any binaries, so `command-line-utilities` also doesn't really fit.
```suggestion
categories = ["development-tools::build-utils"]
```